### PR TITLE
feat: change data_hash type to string and fix get secret key url

### PR DESCRIPTION
**Pull request by gyuguen**
## Implementation
* Change data_hash type of key.proto from byte array to string
* Fixed invalid URL in key.proto

### CODE REVIEW

File: server/service/key/service_key.go [hunk 1]
```diff
@@ -51,7 +51,11 @@ func (s *secretKeyService) GetSecretKey(ctx context.Context, req *key.GetSecretK
 
 	sharedKey := crypto.DeriveSharedKey(oraclePrivKey, consumerPubKey, crypto.KDFSHA256)
 
-	secretKey := GetSecretKey(oraclePrivKey.Serialize(), dealID, req.DataHash)
+	dataHashBz, err := hex.DecodeString(req.DataHash)
+	if err != nil {
+		fmt.Errorf("failed to decode dataHash(%s). %w", req.DataHash, err)
```
Comment (audtlr24) [inline server/service/key/service_key.go:56]: I think you're missing a return statement :)

**Review by audtlr24 — APPROVED** [review]
LGTM!

File: server/service/key/service_key.go (hunk 1 above)
Comment (gyuguen) [reply server/service/key/service_key.go]: OK. I fixed it.(https://github.com/medibloc/panacea-oracle/pull/94/commits/56e5ca3d0b078fbded9416adf60f6098714387ba) 
Thanks

**Review by H4NLee — APPROVED** [review]
lgtm

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Pull request merged**